tests/test_visitors/test_ast/test_classes/test_class_attributes/test_mutable_attributes.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.best_practices import (
+    MutableClassAttributesViolation,
+)
+from wemake_python_styleguide.visitors.ast.classes import (
+    ClassMutableAttributeVisitor,
+)
+
+class_template = """
+class ClassWithAttr(object):
+    field = {0}
+"""
+
+mutable_assign = (
+    '[]',
```

Comment:
We also need to test:
- `set()`, `{1, 2}`
- and all comprehensions

Reply:
i didn't forbid `set`. what to do with `previous_codes`?

Reply:
Oh, yaiks! 

They should really be `frozenset`s 💯

Reply:
done